modules/indexer/code/indexer_test.go

```diff
@@ -96,11 +96,10 @@ func TestBleveIndexAndSearch(t *testing.T) {
 	idx := bleve.NewIndexer(dir)
 	_, err := idx.Init(context.Background())
 	if err != nil {
-		assert.Fail(t, "Unable to create bleve indexer Error: %v", err)
 		if idx != nil {
 			idx.Close()
 		}
```

Comment:
Is this check necessary? I thought `defer idx.Close()` will release it.

Reply:
But defer is in line 105

Reply:
Well, that's true. Never mind.